sys/include/div.h

```diff
@@ -33,17 +33,42 @@ extern "C" {
 /**
  * @brief Approximation of (2**l)/d for d=15625, l=12, 32 bits
  */
-#define DIV_H_INV_15625_32    0x431bde83ul
+#define DIV_H_INV_15625_32     0x431bde83ul
```

Comment:
suggestion put braces and capitalize constant type specification, i.e.
```
#define DIV_H_INV_15625_32     (0x431bde83UL)
```

IMHO using uppercase reads better, especially when defining HEX values.

Reply:
It doesn't matter if you have parentheses or not here, these are just magic numbers and never any complex expressions. As for the letter case, I don't care either way, I tend to prefer lowercase for the suffixes, but I don't see any point in forcing it.